checker-util/src/main/java/org/checkerframework/checker/nullness/util/NullnessUtil.java

```diff
@@ -67,6 +67,7 @@ private NullnessUtil() {
      * @return the argument, casted to have the type qualifier @NonNull
      */
     @EnsuresNonNull("#1")
```

Comment:
suppress 2 warnings in this file

Reply:
After analyzing this file, we think reporting a `cast.unsafe` is a false positive. Not the fp is eliminated.